docs/content/community/doc-system/index.md

```diff
@@ -1,338 +0,0 @@
----
```

Comment:
@tylerbutler I have (for now) removed most of the draft content that was under `docs` previously. I believe you wrote some of this. Is there a place we should copy any of these contents to to preserve them? Or should they be removed.